src/main/java/core/screens/GameScreen.java

```diff
@@ -49,8 +57,16 @@ public void render(float delta) {
         handleUserInput();
         camera.update();
 
+
+
         renderer.setView(camera);
         renderer.render();
+
+        game.batch.setProjectionMatrix(hud.stage.getCamera().combined);
+
+        hud.stage.draw();
```

Comment:
should be moved into a draw method for the whole HUD which the GameplayController can simply call

Reply:
Right, addressed.